server/form-pages/apply/risks-and-needs/risk-to-self/riskToSelfGuidance.ts

```diff
@@ -68,18 +69,21 @@ export default class RiskToSelfGuidance implements TaskListPage {
     let oasysSections
     let taskDataJson
 
-    try {
-      oasysSections = await dataServices.personService.getOasysSections(token, application.person.crn)
-
-      taskDataJson = JSON.stringify(RiskToSelfGuidance.getTaskData(oasysSections))
-    } catch (e) {
-      if (e.status === 404) {
-        oasysSections = null
-      } else {
-        throw e
+    if (!application.data['risk-to-self'] || Boolean(Object.keys(application.data['risk-to-self']).length < 1)) {
```

Comment:
just for my own confusion, what case is the second condition covering here?

Reply:
ah good spot, it doesn't need to be in there, it was just an issue with how integration tests were setup (setting the `risk-to-self` data as empty object, when in real life it wouldn't exist at all)